extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -118,7 +118,7 @@ describes.realWin(
         win
           .getComputedStyle(ampStorySlider.getRootElement())
           .getPropertyValue('--percentage')
-      ).to.be.equal('30%');
+      ).to.be.equal('calc(0.3*100%)');
```

Comment:
Didn't know that you needed to check the property with `calc`, does it fail otherwise? (or can you check directly for the value of `--fraction` instead?)